README.md

```diff
@@ -48,17 +48,36 @@ Add the following content to the `Cargo.toml` file:
 open-feature = "0.1.2"
 ```
 <!-- x-release-please-end -->
-
 ### Usage
 
-#### Basic Usage
+```rust
+async fn example() {
+    // Acquire an OpenFeature API instance.
+    let mut api = OpenFeature::singleton_mut().await;
+
+    // configure a provider
+    api.set_provider(NoOpProvider::builder().bool_value(true).build())
+        .await;
+
+    // create a client
+    let client = api.create_client();
+
+    // get a bool flag value
+    let is_feature_enabled = client
+        .get_bool_value("v2_enabled", None, None)
+        .await
+        .unwrap_or(false);
+}
+```
```

Comment:
We use this exact same example (flag key / type) in all SDKs. Please make sure I haven't made any silly mistakes, my Rust knowledge is approaching zero.

Reply:
```suggestion
```rust
async fn example() -> Result<(), Error> {
    // Acquire an OpenFeature API instance.
    let mut api = OpenFeature::singleton_mut().await;

    // configure a provider
    api.set_provider(NoOpProvider::new())
        .await;

    // create a client
    let client = api.get_client();

    // get a bool flag value
    let is_feature_enabled = client
        .get_bool_value("v2_enabled", false, None)
        .await;

    Ok(())
}
```
```